.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -2,14 +2,14 @@
 # https://github.com/firebase/firebase-tools
 
 name: Deploy to Firebase Hosting on PR
-'on': pull_request
+on: pull_request
```

Comment:
I've always see that `on` with quotes, I'm not sure we should remove it. Does it work?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on

This is a standard YAML syntax. YAML only requires quotes in a very limited number of scenarios, i.e. when string contains special characters.